src/object_.d

```diff
@@ -253,6 +259,16 @@ class TypeInfo
         auto ti = cast(const TypeInfo)o;
         return ti && this.toString() == ti.toString();
     }
+    bool opEquals(const TypeInfo ti)
```

Comment:
I thought about `final` but since the generic opEquals wasn't final I just followed suite.  I guess it really depends on if anyone would want to override the opEquals method.  What do you think?